source/sonata_tech.rst

```diff
@@ -590,8 +590,8 @@ The format of the weights file is described below:
   /electrodes/{electrodename}                   layer                   utf8       1                   Optional      Layer of the circuit in which {electrodename} is located. If the electrode is in a region without cortical layers, then "NA". If the electrode is outside the brain, then "Outside"
   /electrodes/{electrodename}                   position                float32    3                   Mandatory     Position of {electrodename} in microns, in cartesian coordinates
   /electrodes/{electrodename}                   region                  utf8       1                   Optional      Region in which {electrodename} is located
-  /electrodes/{electrodename}                   type                    utf8       1                   Optional      Either EEG or LFP
-  /electrodes/{electrodename}/{population_name} index                   uint64     1                   Mandatory     Index of the column corresponding to this electrode in /electrodes/{population_name}/scaling_factors
+  /electrodes/{electrodename}                   type                    utf8       1                   Optional      Either Reciprocity or LineSource
```

Comment:
can you define what `Reciprocity` and `LineSource` mean?

Reply:
These refer to the different equations used to calculate the coefficients. The line-source equation is an approximation for homogeneous tissue, while the reciprocity equation is more precise. I've added a more detailed explanation in the latest commit

Reply:
Thanks.